### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by xvilo**

### CODE REVIEW

**Comment by ferrydeboer** [comment]
How does this actually behave? And why this version? Afaik the production environment is running 7.3 on Azure.

**Comment by xvilo** [comment]
I just saw I made a misstake, I will fix this later today

**Comment by xvilo** [comment]
So I fixed it now, this change has 2 seperate things:

- The php version `require` (7.3.x)
    - This means, composer will give an error when installing this on another version (can be ignored with `composer install --ignore-platform-reqs`
- The config php version (7.3.0)
    - This will make sure, we will always install/use packages for this specific php version as I could be running php 7.4 on my development machine and installed it with `--ignore-platform-reqs`.

In short, it will make sure we have everything configured and correct for this version.
